src/main/java/org/influxdb/dto/BatchPoints.java

```diff
@@ -184,6 +201,20 @@ void setPoints(final List<Point> points) {
     this.points = points;
   }
 
+  /**
+   * @return the time precision unit
+   */
+  public TimeUnit getPrecision() {
+    return precision;
+  }
+
+  /**
+   * @param precision the time precision to set for the batch points
+   */
+  public void setPrecision(final TimeUnit precision) {
```

Comment:
Why is this setter public, we have a builder which is responsible to do the object creation ?

Reply:
well, seems this long-standing PR has not attracted its author anymore. Let me see if I can help something